internal/dmap/atomic.go

```diff
@@ -79,6 +81,10 @@ func (dm *DMap) atomicIncrDecr(cmd string, e *env, delta int) (int, error) {
 		pool.Put(valueBuf)
 	}()
 
+	if ttl != 0 {
+		e.putConfig.HasEX = true
```

Comment:
@buraksezer do you think `e.putConfig.HasPX` and `e.putConfig.PX` might be a better choice here?

Reply:
@hasit yes, it would be a better choice. Actually, I added `EX` and `PX` commands separately just because of Redis compatibility.

Reply:
[faae5b5](https://github.com/buraksezer/olric/pull/172/commits/faae5b5006a3045e247f6d929d33a12ef71686e9) addresses this.